cdc/owner.go

```diff
@@ -299,9 +306,16 @@ func (c *changeFeed) applyJob(job *pmodel.Job) error {
 	return nil
 }
 
+// procInfoSnap holds most important replication information of a processor
+type procInfoSnap struct {
+	cfID      string
+	captureID string
+	tables    []*model.ProcessTableInfo
+}
+
 type ownerImpl struct {
 	changeFeeds       map[model.ChangeFeedID]*changeFeed
-	markDownProcessor map[string]struct{}
+	markDownProcessor []*procInfoSnap
```

Comment:
Why change it to be a `slice`? What's the implication here?

Reply:
The origin map is not correct since there may exist multiple `changefeeds`